common/string_calls.c

```diff
@@ -172,6 +172,23 @@ g_get_display_num_from_display(const char *display_text)
 
     return rv;
 }
+/*****************************************************************************/
+void
+g_guid_to_string(char *dest, const char *src)
+{
+    unsigned char guid[16] = { 0 };
+
+    for (int i = 0; i < 16 ; i++)
+    {
+        guid[i] = (unsigned char) src[i];
+    }
+
+    g_sprintf(dest, "%02X%02X%02X%02X-%02X%02X-%02X%02X-%02X%02X%02X%02X%02X%02X%02X%02X",
+              guid[3], guid[2], guid[1], guid[0],
+              guid[5], guid[4],
+              guid[7], guid[6],
+              guid[8], guid[9], guid[10], guid[11], guid[12], guid[13], guid[14], guid[15]);
+}
```

Comment:
Normal textual representation of a `UUID` is 8-4-4-4-12 rather than 8-4-4-16 (see [here](https://en.wikipedia.org/wiki/Universally_unique_identifier) and also [[MS-DTYP] 2.3.4.3](https://learn.microsoft.com/en-us/openspecs/windows_protocols/ms-dtyp/222af2d3-5c00-4899-bc87-ed4c6515e80d)). Suggest we use that instead as it's slightly more readable.

Reply:
Agreed.